scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -151,6 +151,51 @@ namespace <%= namespace %> {
             } catch(NoQueryException exception) {}
         }
 
+        private void SetShippingAddress(MailingAddressInput mailingAddressInput, Action<List<string>, string> callback) {
+            MutationQuery query = new MutationQuery();
+
+            DefaultQueries.checkout.ShippingAddressUpdate(query, CurrentCheckout.id(), mailingAddressInput);
+
+            Client.Mutation(query, (Mutation response, List<string> errors, string httpError) => {
+                if (errors != null || httpError != null) {
+                    callback(errors, httpError);
+                } else {
+                    UpdateCheckout(response.checkoutShippingAddressUpdate().checkout());
+                    AvailableShippingRatesPoll(callback);
+                }
+            });
+        }
+
+        private void SetShippingLine(string shippingRateHandle, Action<List<string>, string> callback) {
```

Comment:
Yes but it seems to be a refactoring effort. I think it would be better if it went into https://github.com/Shopify/unity-buy-sdk/pull/110. So we don't repeat ourselves